codecov_cli/commands/upload.py

```diff
@@ -59,9 +59,24 @@ def _turn_env_vars_into_dict(ctx, params, value):
         multiple=True,
         default=[],
     ),
+    click.option(
+        "--test-file",
```

Comment:
I think you should hold on adding these options until we can actually process the data.
It's gonna be useless for users to have this if they can't use it, and possibly confusing as well